jupyter_book/book_template/_data/toc.yml

```diff
@@ -85,6 +85,13 @@
     url: /advanced/netlify
     not_numbered: true
 
+- title: Beta Features
```

Comment:
I find it a little confusing both to have this section _and_ all of the above sections explicitly marked `BETA`. Are you planning to consolidate these ?

Reply:
Yep, these have been consolidated on Master and I'll rebase and fix this one on this PR